pkg/cmd/registry/artifact/crud/create/create.go

```diff
@@ -73,6 +73,12 @@ func NewCreateCommand(f *factory.Factory) *cobra.Command {
 				opts.file = args[0]
 			}
 
+			if opts.artifactType != "" {
+				if _, err := registryinstanceclient.NewArtifactTypeFromValue(opts.artifactType); err != nil {
+					return errors.New(opts.localizer.MustLocalize("artifact.cmd.create.error.invalidArtifactType", localize.NewEntry("AllowedTypes", util.GetAllowedArtifactTypeEnumValuesAsString())))
```

Comment:
```suggestion
					return errors.New(opts.localizer.MustLocalize("artifact.cmd.create.error.invalidArtifactType", localize.NewEntry("AllowedTypes", util.GetAllowedArtifactTypeEnumValuesAsString())))
```
```suggestion
					return opts.localizer.MustLocalizeError("artifact.cmd.create.error.invalidArtifactType", localize.NewEntry("AllowedTypes", util.GetAllowedArtifactTypeEnumValuesAsString()))
```

Reply:
Good catch. Will fix that in couple places.